sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -163,6 +215,18 @@ def _auto_lock_renew(
                         # Renewable is a message
                         receiver.renew_message_lock(renewable)  # type: ignore
                 time.sleep(self._sleep_time)
```

Comment:
I know this is not new - but why was the sleep necessary here?

Reply:
as oob discussion: leaving it here to save CPU cycles